src/vs/workbench/contrib/terminal/browser/terminalInstance.ts

```diff
@@ -1160,9 +1161,6 @@ export class TerminalInstance extends Disposable implements ITerminalInstance {
 		this._pressAnyKeyToCloseListener?.dispose();
 		this._pressAnyKeyToCloseListener = undefined;
 
-		// Kill and clear up the process, making the process manager ready for a new process
-		this._processManager.dispose();
-
```

Comment:
Did this need to be moved?

Reply:
It was just so that that one is next to the line that creates the new one. More readable.

Reply:
It's probably fine, but this may reorder when some events happen. Need to be careful of regressions.